pkg/controller/stack/stack_controller.go

```diff
@@ -262,17 +261,27 @@ func (r *ReconcileStack) Reconcile(ctx context.Context, request reconcile.Reques
 	// If a branch is specified, then track changes to the branch.
 	trackBranch := len(sess.stack.Branch) > 0
 
+	resyncFreqSeconds := sess.stack.ResyncFrequencySeconds
+	if sess.stack.ResyncFrequencySeconds != 0 && sess.stack.ResyncFrequencySeconds < 60 {
+		resyncFreqSeconds = 60
+	}
```

Comment:
Any particular reason to set 60 seconds as the minimum? I think it's a reasonable default, but it seems like we could allow users to opt into a shorter interval.

Reply:
I am a little concerned about the overhead of having to checkout and then cleanup resources around each sync. There is probably some low-lift caching we can do here but given current use cases 60 seconds seems like an acceptable limit to place for now.